lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -671,6 +671,23 @@ sub find_for_cdtoc
     $self->query_to_list_limited($query, [$track_count, $artist_id], $limit, $offset);
 }
 
+sub find_gid_for_medium
+{
+    my ($self, $medium_id) = @_;
+
+    # A medium is not a user visible entity, this function is called by
+    # the medium controller to issue a redirect to the release page
+    # on which the medium appears. So only the release MBID is needed.
+
+    my $query =
```

Comment:
this can use `<<~'SQL'` (not a big deal, but it makes the indentation nicer when queries are logged)